Sources/Common/TimeoutError.swift

```diff
@@ -27,8 +27,8 @@ public struct TimeoutError: Error, LocalizedError, CustomDebugStringConvertible
     public let file: StaticString
     public let line: UInt
 
-    // swiftlint:disable:next line_length
     public var errorDescription: String? {
+        // swiftlint:disable:next line_length
```

Comment:
Would rather disable this than mess with wrapping ultra long strings like this one